### DISCUSSION TITLE
What is the best approach to run regularly another github workflow to create the PR for the update?

### DISCUSSION THREAD

**Discussion by pdonorio** [renovatebot/renovate#25618] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Since we are already using renovate for all kind of updates,
and we like the opportunity to centralize the bump logic in one place,
we were wondering how to achieve the following use case within renovate:

* running once a week
* run a dedicated command in python to create a full update
* present the PR with the update results in a single commit

Renovate itself shouldn't do anything in particular.

The dedicated command is already "packaged" in another github workflow, so either running the command or triggering the workflow would work, with a preference to trigger the existing workflow as it's already there.

### What you tried so far.

We've seen https://docs.renovatebot.com/configuration-options/#postupgradetasks but it seems to be something to run after renovate does the upgrade, while in this case renovate would just be handling the call of the command, not do anything itself.

Any direction/pointer would be much appreciated, thanks a lot for this useful tool!

**Comment by pdonorio** [comment]
Can I improve the question by clarifying something maybe?

**Comment by rarkins** [comment, marked as answer]
We are not planning to evolve Renovate into a "PR anything" solution which is what would be required to satisfy what you're asking

**Comment by pdonorio** [comment]
Thanks for replying 👍
